pkg/controller/operands/kubevirt.go

```diff
@@ -245,16 +245,20 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1
 		uninstallStrategy = kubevirtcorev1.KubeVirtUninstallStrategyRemoveWorkloads
 	}
 
+	kv := NewKubeVirtWithNameOnly(hc, opts...)
+
 	spec := kubevirtcorev1.KubeVirtSpec{
 		UninstallStrategy:           uninstallStrategy,
 		Infra:                       hcoConfig2KvConfig(hc.Spec.Infra, infrastructureHighlyAvailable),
 		Workloads:                   hcoConfig2KvConfig(hc.Spec.Workloads, true),
 		Configuration:               *config,
 		CertificateRotationStrategy: *kvCertConfig,
 		WorkloadUpdateStrategy:      hcWorkloadUpdateStrategyToKv(hc.Spec.WorkloadUpdateStrategy),
+		ProductName:                 hcoutil.HyperConvergedCluster,
+		ProductVersion:              os.Getenv(hcoutil.HcoKvIoVersionName),
+		ProductComponent:            kv.ObjectMeta.Labels[hcoutil.AppLabelComponent],
```

Comment:
You can set `hcoutil.AppComponentCompute` because this is the constant value we set on `kv.ObjectMeta.Labels[hcoutil.AppLabelComponent]`

Reply:
Thank you for reviewing.
Done. :)